src/index.html

```diff
@@ -7,6 +7,69 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="calendar calendar--start-day--mon calendar--month-length--31">
+      <div class="calendar__day calendar__day--1"></div>
+
```

Comment:
```suggestion
      <div class="calendar__day calendar__day"></div>
```

Reply:
please read requirements:
![image](https://github.com/mate-academy/layout_calendar/assets/102289654/dd65abca-09f6-4008-bb8b-d236a3ac4cea)
and remove indent between days as it is unnecessary